clusters/hive/pools/hypershift/hypershift-ocp-4-9-0-amd64-aws-us-east-1_clusterpool.yaml

```diff
@@ -1,33 +1,38 @@
 apiVersion: hive.openshift.io/v1
 kind: ClusterPool
 metadata:
-  name: hypershift-ocp-4-9-amd64-aws-us-east-1
-  namespace: hypershift-cluster-pool
+  creationTimestamp: null
   labels:
-    product: ocp
-    version: "4.9"
     architecture: amd64
     cloud: aws
     owner: hypershift
+    product: ocp
     region: us-east-1
+    version: "4.9"
+    version_lower: 4.9.0-0
+    version_upper: 4.10.0-0
+  name: hypershift-ocp-4-9-amd64-aws-us-east-1
+  namespace: hypershift-cluster-pool
 spec:
   baseDomain: hive.hypershift.devcluster.openshift.com
   imageSetRef:
-    name: ocp-release-4.9.0-rc.1-x86-64-for-4.9.0-0-to-4.10.0-0
+    name: ocp-release-4.9.22-x86-64-for-4.9.0-0-to-4.10.0-0
   installAttemptsLimit: 3
   installConfigSecretTemplateRef:
     name: install-config-aws-us-east-1
   labels:
     hive.openshift.io/resume-skips-cluster-operators: "true"
-  skipMachinePools: true
+  maxSize: 12
   platform:
     aws:
       credentialsSecretRef:
         name: aws-creds-osd-hypershift
       region: us-east-1
   pullSecretRef:
     name: pull-secret
+  size: 12
+  skipMachinePools: true
+status:
+  ready: 0
   size: 0
-  maxSize: 4
-  hibernationConfig:
-    resumeTimeout: 20m
```

Comment:
Weird, why did this get removed? This should be used to not block an entire pool on one broken clusterdeployment that doesn't resume (but we can check in a follow-up)